src/go/k8s/kind.yaml

```diff
@@ -2,20 +2,20 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-  image: vectorizedcloud/kindestnode:v1.21.1-pr2515
```

Comment:
These changes are happening on another PR, Let's the change here for now and rebase when : https://github.com/redpanda-data/redpanda/pull/8310/files is merged in.

Reply:
Yes, that's right. I will rebase once https://github.com/redpanda-data/redpanda/pull/8310 is merged.

I mentioned it in cover letter of this PR